### PR TITLE
Use return value from std::unique.

### PR DESCRIPTION
**Pull request by toregge**
@vekterli : please review

### CODE REVIEW

File: vdslib/src/tests/distribution/distributiontest.cpp [hunk 1]
```diff
@@ -434,7 +434,8 @@ TEST(DistributionTest, test_distribution)
                 _distribution[i].second = distr.getIdealStorageNodes(
                         systemState, document::BucketId(26, i));
                 sort(_distribution[i].second.begin(), _distribution[i].second.end());
-                unique(_distribution[i].second.begin(), _distribution[i].second.end());
+                auto unique_nodes = unique(_distribution[i].second.begin(), _distribution[i].second.end()) - _distribution[i].second.begin();
```
Comment (vekterli) [inline vdslib/src/tests/distribution/distributiontest.cpp:437]: This might benefit from using `std::distance` instead, to make the semantics more clear

**Review by vekterli — DISMISSED** [review]
:+1:

File: vdslib/src/tests/distribution/distributiontest.cpp (hunk 1 above)
Comment (toregge) [reply vdslib/src/tests/distribution/distributiontest.cpp]: Done.

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**